### ISSUE TITLE
[Feature-Request] Support for German letters

### ISSUE THREAD

**Issue by CommanderStorm** [just-max/exzellenz#7]
It would be really nice for this tool to support `äöü` and `ß`. 

I know, that this would probably be breaking the max-height rule, but this would be acceptable

**Comment by just-max** [comment]
Yes, certainly possible. The generator page would need to be updated, since it assumes a constant glyph height. I think this would be easy to account for using opentype.js, which would also enable exporting as an SVG path instead of text.

I'll put it on my TODO list for the next time I'm procrastinating something else!
